docs/source/devguides/walkthroughs/DrivenMatlabAgent-Walkthrough.rst

```diff
@@ -110,13 +196,90 @@ Steps to test integration:
 
 3. Run device driver agent
 
-4. Run DrivenMatlabAgent with the new config file
+4. Install  DrivenMatlabAgent with the new config file
+
+.. code-block:: console
+
+   python scripts/install-agent.py  -s <source
+   directory of volttron-applications>/pnnl/DrivenMatlabAgent -c
+   config/config_waterheater
+
+5. Run DrivenMatlabAgent
```

Comment:
vctl start --name DrivenMat*

will work to start it...or you can put --start --vip-identity matlab --force to make sure there is only one installed.